mautic/mautic
# Enable Slave Connections if exists

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | X
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | [mautic/developer-documentation#...](https://github.com/mautic/developer-documentation-new/pull/104) 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Right now there is method to split the connection between a master an slave databases, this PR solve this issue by splitting the connections to read-only and read-write connections



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Modify your `local.php` file to include a connection to a remote database that has replica configured, add the following parameter ```'db_host_ro'=>'db'``` and change db with the name of the read replica of your database
3. Make sure that the read replica is getting queries
